cmpserver/plugin/plugin.go

```diff
@@ -106,14 +105,35 @@ func runCommand(ctx context.Context, command Command, path string, env []string)
 	logCtx.WithFields(log.Fields{"duration": duration}).Debug(output)
 
 	if err != nil {
-		err := newCmdError(args, errors.New(err.Error()), strings.TrimSpace(stderr.String()))
+		err := newCmdError(argsToLog, errors.New(err.Error()), strings.TrimSpace(stderr.String()))
 		logCtx.Error(err.Error())
 		return strings.TrimSuffix(output, "\n"), err
 	}
 
 	return strings.TrimSuffix(output, "\n"), nil
 }
 
+// getCommandArgsToLog represents the given command in a way that we can copy-and-paste into a terminal
+func getCommandArgsToLog(cmd *exec.Cmd) string {
+	var argsToLog []string
+	for _, arg := range cmd.Args {
+		if strings.ContainsRune(arg, ' ') {
```

Comment:
Would it maybe better to loop over each rune in the string and use `unicode.IsSpace` here?

Reply:
What would be the advantage? (I know basically nothing about strings, runes, and unicodes. 😛)

Reply:
It would catch white space characters other than a plain space character (such as, a tab stop, line breaks, etc). Don't know how applicable that is in the plugin configuration, though. So I guess while it would be "more correct", it doesn't necessarily make "more sense" here :thinking: